src/main/java/com/mael/payroll/paymentTypes/HourlyPayment.java

```diff
@@ -10,4 +13,12 @@ public HourlyPayment(double hourlyRate) {
     public double getHourlyRate() {
         return hourlyRate;
     }
+
+    public TimeCard getTimeCard() {
+        return timeCard;
+    }
+
+    public void addTimeCard(TimeCard timeCard) {
```

Comment:
This is nitpicking, but `addTimeCard()` gives me the impression I could add more than one time card to the payment. As in "I'm adding something to a list".
Is there a specific reason why you used `add` instead `set`?